packages/@lwc/babel-plugin-component/src/post-process/dedupe-imports.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
```

Comment:
nit picking: is 2018 correct?

Reply:
usually we use the year of the file creation.